src/Interfaces/index.ts

```diff
@@ -16,6 +16,9 @@ export interface IFormValues {
   phoneNumber?:number
   profileImage?:string
   code?:string
+  title?:string
```

Comment:
You can make all of them required and use the Pick utility to transform from it different interfaces, and this make this interface the only source of truth if you wanted to change any type later

something like that
![image](https://github.com/mohamedsaeid1998/Project-Management-System/assets/64327685/6785d493-ae00-4fc6-be9d-28e7b3376f46)

you can read more about the Pick utility [here](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys)